samples/bookinfo/src/productpage/Dockerfile

```diff
@@ -15,11 +15,14 @@
 FROM python:3.7.7-slim
 
 WORKDIR /
+RUN apt-get update \
+&& apt-get install make gcc -y \
```

Comment:
do we need this?

Reply:
yah to build `gevent`

Reply:
Is it only for cross compiling it? Seems to work from me, it pulls from precompiled? 

Adding these will bloat the images quite a bit

Reply:
yes the issues show up while building the `linux/arm64` image

Reply:
the image size has doubled to 121 MB (https://hub.docker.com/r/arkodg/examples-bookinfo-productpage-v1/tags) from 63 MB (https://hub.docker.com/r/istio/examples-bookinfo-productpage-v1/tags)

Reply:
It is old gevent, newer one is fine. Can we update it? 

We might want to update all our deps, they seem super old. But I don't want to scope creep

Reply:
> It is old gevent, newer one is fine. Can we update it?
> 

cool, let me try

Reply:
works !